cloudformation/xtdb_s3.yml

```diff
@@ -0,0 +1,100 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: A stack for setting up XTDB S3 object store and associated resources (SNS topics for changes, XTDB user)
+
+Parameters:
+  S3BucketName:
+    Type: String
+    Default: xtdb-cluster-bucket
```

Comment:
We can't really give a default for the XTDB bucket name, as these have to be globally unique - i.e. as soon as one user uses the default nobody else can